### PR TITLE
CS-9 deploy to multiple nuget packages

### PR DESCRIPTION
**Pull request by mdallago**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -13,49 +17,25 @@ environment:
...
-        cmd.exe /c "dotnet build $env:APPVEYOR_BUILD_FOLDER\bitprim\bitprim.csproj /property:Platform=x64"
-        cmd.exe /c "dotnet nuget push $env:APPVEYOR_BUILD_FOLDER\bitprim\bin\x64\Release\*.nupkg -s https://api.nuget.org/v3/index.json -k $env:NUGET_API_KEY"
-      }
-      Get-Content $env:APPVEYOR_BUILD_FOLDER\bitprim.tests\bin\x64\Release\netcoreapp2.0\bitprim-cs-log.txt
+build_script:
+  - SET PATH=%PATH%;%PYTHON%/Scripts/
+  - powershell .\build.ps1
+
+test: off
+#skip_tags: true
```
Comment (dario-ramos) [inline appveyor.yml]: Remove dead code

File: bitprim.tests/bitprim.tests.csproj [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <Import Project="..\bitprim\build\Common.targets"/>
+  <Import Project="..\bitprim-bch\build\Common.targets" />
```
Comment (dario-ramos) [inline bitprim.tests/bitprim.tests.csproj]: Tests should be run for btc too

File: build.cake [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+        DotNetCorePack(bitprimProjectBCH, settings);
+        DotNetCorePack(bitprimProjectBTC, settings);
+
+        System.IO.File.WriteAllLines(outputDir + "artifacts", new[]{
+            "nuget:bitprim-bch." + versionInfo.NuGetVersion + ".nupkg",
+            "nuget:bitprim-btc." + versionInfo.NuGetVersion + ".nupkg"
+        });
+
+        //if (AppVeyor.IsRunningOnAppVeyor)
```
Comment (dario-ramos) [inline build.cake]: Remove dead code

File: build.cake [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+        var files = System.IO.File
+            .ReadAllLines(outputDir + "artifacts")
+            .Select(l => l.Split(':'))
+            .Select(l => l[1])
+            .ToList();
+
+        foreach (string f in files)
+        {
+            Information("Pushing to nuget " + f);
+           /* NuGetPush(
```
Comment (dario-ramos) [inline build.cake]: Remove dead code

Comment (mdallago) [reply build.cake]: it's commented temporarily to avoid publish to nuget

**Pull request merged**